### DISCUSSION TITLE
Welcome to dynetx Discussions!

### DISCUSSION THREAD

**Discussion by GiulioRossetti** [GiulioRossetti/dynetx#88] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by bibrakc** [comment]
Is there a way to generate a dynamic graph just like we do in networkx but this time we call the generator multiple times on the same graph and each time it adds a set number of edges?
